Dockerfile

```diff
@@ -5,11 +5,9 @@ ENV RACK_ENV=development
 WORKDIR /usr/src/app
 
 COPY Gemfile Gemfile.lock .ruby-version ./
-RUN apk --update --upgrade add build-base mysql-dev && \
-  bundle check || ${BUNDLE_INSTALL_CMD} && \
-  apk del build-base && \
-  find / -type f -iname \*.apk-new -delete && \
-  rm -rf /var/cache/apk/*
+RUN apk --no-cache add build-base mysql-dev && \
```

Comment:
Even better.. `apk --no-cache add --virtual .build-deps build-base mysql-dev && ${BUNDLE_INSTALL_CMD} && apk del .build-deps`

Might need to explicitly `apk add` the mysql libs so they don't get purged by the `apk del`